src/components/genre-select/genre-select.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './genre-select.scss';
+
+export default class GenreSelect extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      selection: this.props.list[0],
+      list: this.props.list
+    };
+
+    this.select = this.select.bind(this);
```

Comment:
The same questions as for Search-form component

Reply:
fixed in https://github.com/SiarheiDrozd/ReactJS/pull/5